src/app/helpers/jwt.interceptor.ts

```diff
@@ -1,16 +1,22 @@
-import {Injectable} from '@angular/core';
-import {HttpEvent, HttpHandler, HttpInterceptor, HttpRequest} from '@angular/common/http';
-import {Observable} from 'rxjs';
+import { Injectable } from '@angular/core';
+import { HttpEvent, HttpHandler, HttpInterceptor, HttpRequest } from '@angular/common/http';
+import { Observable } from 'rxjs';
+import { AuthService } from '../../app/services/auth.service';
```

Comment:
@Manorlds-Eaglespark , we can use `absolute path` here

Reply:
This has been resolved, kindly take another look

Reply:
not yet @Manorlds-Eaglespark

Reply:
Please check, thanks @Tevinthuku